Dockerfile

```diff
@@ -56,6 +15,8 @@ RUN /bin/bash -c "source /opt/ros/$ROS_DISTRO/setup.bash; colcon build"
 # change to the home root
 WORKDIR ${HOME}
 
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/openrobots/lib/
```

Comment:
Is this needed again? its already done in the install script

Reply:
This is related to the problem that the libraries must be installed as `root` in `modulo`. When using `sudo` the environment variables are not set. This is the only hack I have found to make the compilation work.

Reply:
why are you not root here? I guess because you base on your own ros2_ws image where you created a user at the end?

Reply:
yes